test/odbc/psqlodbc/test_pg_varchar.cpp

```diff
@@ -0,0 +1,655 @@
+#include <gtest/gtest.h>
+#include <sqlext.h>
+#include "../src/drivers.h"
+#include "../src/odbc_handler.h"
+#include "../src/query_generator.h"
+#include "psqlodbc_tests_common.h"
+#include <string.h>
+#include <iostream>
+using std::pair;
+
+const string TABLE_NAME = "master_dbo.varchar_table_odbc_test";
+const string VIEW_NAME = "master_dbo.varchar_view_odbc_test";
+const string COL1_NAME = "pk";
+const string COL2_NAME = "varchar_1";
+const string COL3_NAME = "varchar_8000";
+const string COL4_NAME = "varchar_20";
+
+const string DATATYPE = "sys.varchar";
+
+vector<pair<string, string>> TABLE_COLUMNS_1 = {
+  {COL1_NAME, " int PRIMARY KEY"},
+  {COL2_NAME, DATATYPE + "(1)"}
+};
```

Comment:
What about a table without specifying `n`
```
ie 
```
vector<pair<string, string>> TABLE_COLUMNS_1 = {
  {COL1_NAME, " int PRIMARY KEY"},
  {COL2_NAME, DATATYPE}
};
```

Reply:
This will set as default value, for PG it's max value, for BBF it's min value